nbviewer/providers/local/handlers.py

```diff
@@ -115,6 +115,7 @@ def can_show(self, path):
             return False
 
         if not os.path.exists(fullpath):
+            app_log.warn("path: '%s' does not exist" % fullpath)
```

Comment:
Thanks for the PR. I think adding all of these should be fine. Could you please change them all to use the standard Python logging module approach of passing arguments as the second parameter before we merge? 

For example:

`app_log.warn("path: '%s' does not exist", fullpath)`

should work, is standard, and saves the interpolation when the log level is set higher than the function.